src/Symfony/Component/Notifier/Bridge/MicrosoftTeams/Section/AbstractMicrosoftTeamsSection.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\MicrosoftTeams\Section;
+
+/**
+ * @author Edouard Lescot <edouard.lescot@gmail.com>
+ */
+abstract class AbstractMicrosoftTeamsSection implements MicrosoftTeamsSectionInterface
```

Comment:
This is not needed. Only used once.

Reply:
see https://github.com/symfony/symfony/pull/40738#discussion_r611879277